install_config/certificate_customization.adoc

```diff
@@ -61,7 +61,7 @@ openshift_hosted_router_certificate={"certfile": "/path/on/host/to/app-crt-file"
 Example parameters for a master API certificate:
 ----
 openshift_master_overwrite_named_certificates=true
-openshift_master_named_certificates=[{"names": ["master.148.251.233.173.nip.io"], "certfile": "/home/cloud-user/master-bundle.cert.pem", "keyfile": "/home/cloud-user/master.148.251.233.173.nip.io.key.pem", "cafile": "/home/cloud-user/master.148.251.233.173.nip.io.key.pem"}]
+openshift_master_named_certificates=[{"names": ["master.148.251.233.173.nip.io"], "certfile": "/home/cloud-user/master-bundle.cert.pem", "keyfile": "/home/cloud-user/master.148.251.233.173.nip.io.key.pem", "cafile": "/home/cloud-user/master-bundle.cert.pem"}]
```

Comment:
Change "certfile": "/home/cloud-user/master-bundle.cert.pem" to "/home/cloud-user/master.148.251.233.173.nip.io.key.pem".